INSTALL.md

```diff
@@ -25,11 +25,11 @@ source code.  Unpack as follows.
     tar xzf <some dir>/accumulo-X.Y.Z-bin.tar.gz
     cd accumulo-X.Y.Z
 
-There are three scripts in the the `bin/` directory that are used to manage Accumulo:
+There are three scripts in the tarball distribution that are used to manage Accumulo:
 
-1. `accumulo` - Runs Accumulo command-line tools and starts Accumulo processes
-2. `accumulo-service` - Runs Accumulo processes as services
-3. `accumulo-cluster` - Manages Accumulo cluster on a single node or several nodes
+1. `bin/accumulo` - Runs Accumulo command-line tools and starts Accumulo processes
+2. `contrib/accumulo-service` - Runs Accumulo processes as services
```

Comment:
I thought about having a contrib and a bin directory. I think there should only be a bin/ directory.  Two directories will confuse people.  We can communicate that scripts are optional/add-on in the documentation rather than the directory structure.  To limit the number of scripts in bin, we can combine many of the scripts in contrib into one or two scripts.  If a script doesn't work or we don't want to support it any more the in the tarball distribution, it should either be removed or move to a contrib directory in the repo that doesn't get added to the distribution. Contrib scripts should not be distributed.  As for the scripts in bin/, downstream packagers can pick and choose what they include.